src/components/pages/Categories/Management.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <ul class="category">
+    <!-- カテゴリー管理 -->
+    <li class="category-post">
+      <CategoryPost
+        :access="access"
+      />
+    </li>
+    <!-- カテゴリー名 -->
+    <li class="category-list">
+      <CategoryList
+        :theads="theads"
+        :categories="categoryList"
+      />
+    </li>
+  </ul>
+</template>
+
+<script>
+import { CategoryPost, CategoryList } from '@Components/molecules';
+
+export default {
+  components: {
+    CategoryPost,
+    CategoryList,
+  },
+  data() {
+    return {
+      theads: ['カテゴリー名'],
+    };
+  },
+  computed: {
+    access() {
+      return this.$store.getters['auth/access'];
+    },
+    categoryList() {
+      return this.$store.state.categories.categoryList;
+    },
+  },
+  created() {
+    this.$store.dispatch('categories/getCategories');
+  },
+};
+</script>
+<style lang="scss" scoped>
```

Comment:
[nits]

template, script, style タグ間は1行空行を空けると見やすくなるかと思います。


```suggestion
</script>

<style lang="scss" scoped>
```